package.json

```diff
@@ -1,15 +1,14 @@
 {
-  "name": "ts-express-jest",
-  "version": "1.0.0",
```

Comment:
Let's keep the version 0.0.0 till the initial release

Reply:
Sure, I think the PR changes 1.0.0 to 0.0.0 so no changes needed here right?